lib/fastlane/plugin/wpmreleasetoolkit/helper/an_metadata_update_helper.rb

```diff
@@ -134,5 +134,24 @@ def extract_key(line)
       end
     end
 
+    class ReleaseNoteShortMetadataBlock < ReleaseNoteMetadataBlock     
+
+      def initialize(block_key, content_file_path, release_version)
+        super(block_key, content_file_path, release_version)
+        @rel_note_key = "release_note_short"
+        @release_version = release_version
+        generate_keys(release_version)
+      end
+  
+      def is_handler_for(key)
+        values = key.split('_')
+        key.start_with?(@rel_note_key) && values.length == 4 && (Integer(values[3].sub(/^[0]*/, "")) != nil rescue false)
+      end
+
+    end
   end
+
+
+
```

Comment:
These empty lines are not necessary, but I don't want it to hold off the release, so just noting here hoping it'll remind us to remove them when we are working on something else.

Reply:
Thanks Oguz!